test/test_mediawiki.py

```diff
@@ -6,33 +6,20 @@
 
 user = "Test"
 user_id = 0
-fail_user = "fail"
+fail_user = "Fail"
 password = "******"
 
 
 @pytest.fixture
 def db_conn():
     """Set up chimedb.core for testing with a local dummy DB."""
     (fd, rcfile) = tempfile.mkstemp(text=True)
-    with os.fdopen(fd, "a") as rc:
-        rc.write(
-            """\
-        chimedb:
-            db_type:         MySQL
-            db:              test
-            user_ro:         travis
-            passwd_ro:       ""
-            user_rw:         travis
-            passwd_rw:       ""
-            host:            127.0.0.1
-            port:            3306
-        """
-        )
+    os.close(fd)
```

Comment:
Why did this change?

Reply:
So I don't have to create that MySQL db on github ci